create-gcloud-instance/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Create a new linux self-hosted runner GitHub Action
+
+An action that creates a new self-hosted runner using gcloud.
+
+The GITHUB_TOKEN has to have "public repo" access.
+
+## Usage
+
+```yaml
+- name: Create a new linux self-hosted runner
+  uses: Homebrew/actions/create-gcloud-instance@master
+  with:
```

Comment:
Those are not aligned with actual action's inputs.

Reply:
Done

Reply:
`repository_name` is missing